app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Whatsapp.vue

```diff
@@ -117,18 +109,38 @@ const shouldShowCloudWhatsapp = provider => {
     </div>
 
     <div v-else-if="showConfiguration">
-      <WhatsappEmbeddedSignup
-        v-if="shouldShowEmbeddedSignup(selectedProvider)"
-      />
-      <CloudWhatsapp v-else-if="shouldShowCloudWhatsapp(selectedProvider)" />
-      <Twilio
-        v-else-if="selectedProvider === PROVIDER_TYPES.TWILIO"
-        type="whatsapp"
-      />
-      <ThreeSixtyDialogWhatsapp
-        v-else-if="selectedProvider === PROVIDER_TYPES.THREE_SIXTY_DIALOG"
-      />
-      <CloudWhatsapp v-else />
+      <div class="px-6 py-5 rounded-2xl border bg-n-solid-2 border-n-weak">
+        <!-- Show embedded signup if app ID is configured -->
+        <div
+          v-if="
+            hasWhatsappAppId && selectedProvider === PROVIDER_TYPES.WHATSAPP
+          "
+        >
+          <WhatsappEmbeddedSignup />
+
+          <!-- Manual setup fallback option -->
+          <div class="pt-6 mt-6 border-t border-n-weak">
+            <p
+              class="text-sm text-n-slate-11"
+              @click="handleManualLinkClick"
+              v-html="manualFallbackText"
+            />
```

Comment:
This approach isn’t a good one. Using `v-html` for string replacement can introduce security issues, and event handlers on interactive elements won’t work correctly. Instead, let us use [Vue i18n’s Component Interpolation](https://vue-i18n.intlify.dev/guide/advanced/component#component-interpolation) with the `<I18nT>` component and named slots.

For eg:
```
  <I18nT
    keypath="INBOX_MGMT.ADD.WHATSAPP.EMBEDDED_SIGNUP.MANUAL_FALLBACK"
    tag="span"
    class="text-sm text-n-slate-11"
  >
    <template #link>
      <a
        href="#"
        class="underline !text-n-blue-text"
        @click.prevent="handleManualLinkClick"
      >
        {{ $t('INBOX_MGMT.ADD.WHATSAPP.EMBEDDED_SIGNUP.MANUAL_LINK_TEXT') }}
      </a>
    </template>
  </I18nT>
```